fbu_twitter_starter/twitter/Models/Tweet.h

```diff
@@ -0,0 +1,33 @@
+//
+//  Tweet.h
+//  twitter
+//
+//  Created by Abigail Shilts on 6/20/22.
+//  Copyright © 2022 Emerson Malca. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "User.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Tweet : NSObject
+
+@property (nonatomic, strong) NSString *idStr; // For favoriting, retweeting & replying
+@property (nonatomic, strong) NSString *text; // Text content of tweet
+@property (nonatomic, strong) NSString *createdAtString; // Display date
+@property (nonatomic) int favoriteCount; // Update favorite count label
+@property (nonatomic) int retweetCount; // Update favorite count label
+@property (nonatomic) BOOL favorited; // Configure favorite button
+@property (nonatomic) BOOL retweeted; // Configure retweet button
+@property (nonatomic, strong) User *user; // Contains Tweet author's name, screenname, etc.
+
+// For Retweets
+@property (nonatomic, strong) User *retweetedByUser;  // If the tweet is a retweet, this will be the user who retweeted
+
+- (instancetype)initWithDictionary:(NSDictionary *)dictionary;
++ (NSMutableArray *)tweetsWithArray:(NSArray *)dictionaries;
```

Comment:
nit: The best practice is not return NSArray even if NSMutableArray is used under the hood. That way you can guarantee that returned value won't be mutated by other parts of the code.
Also, it's beneficial to use generics with objective c collections. NSArray<Tweet>.